client/views/room/Header/RoomHeader.js

```diff
@@ -31,7 +33,7 @@ const RoomHeader = ({ room, topic }) => {
 			<Header.Content>
 				<Header.Content.Row>
 					<RoomTitle room={room} />
-					<Favorite room={room} />
+					{subscription ? <Favorite room={room} /> : null}
```

Comment:
Hey @shrinish123, i think this maybe could be shortened in:
```suggestion
					{subscription && <Favorite room={room} />}
```
What do you think ?